inginious/frontend/plugins/problem_bank/react_app/src/custom_alert.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Alert } from 'react-bootstrap';
+
+class CustomAlert extends React.Component {
```

Comment:
We are using "Sweet Alet" and Custom Alert. why are we using two different classes with similar functionality

Reply:
They are different, Sweet alert behaves as a modal but only shows alerts and allow some inputs, such as  `alert()` function but prettier.